pkg/flow/reconciler/xmltojsontransformation/controller_test.go

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
Why delete this test?

Reply:
I could not get it to work :/

Reply:
What was failing for it? Note that if this test fails, then that means the adapter will fail to come up.

Reply:
```    /Users/jeffreynaef/go/src/github.com/triggermesh/triggermesh/pkg/flow/reconciler/xmltojsontransformation/logger.go:130: 2022-01-21T09:32:38.863-0700	PANIC	addressable/addressable.go:56	Unable to fetch knative.dev/pkg/apis/duck.InformerFactory from context.
        /Users/jeffreynaef/go/src/github.com/triggermesh/triggermesh/pkg/flow/reconciler/xmltojsontransformation/controller.go:61: Unexpected panic: Unable to fetch knative.dev/pkg/apis/duck.InformerFactory from context.
```